postprocess.ts

```diff
@@ -0,0 +1,38 @@
+import { readCSV, writeCSV, removeFile } from 'https://deno.land/x/flat@0.0.15/mod.ts' // replace with latest library https://deno.land/x/flat@0.0.x/mod.ts
+
+// flat action runs on Deno, a Node competitor. Flat will pass in the downloaded filename as arg 0.
+const filePath: string = Deno.args[0]
+
+// without lazyQuotes some of the values break readCSV
+const tribalLeaders: Record<string, any> = await readCSV(filePath, {lazyQuotes: true})
+
+// custom compare function
+function compare( a, b ) {
+    if ( a["Tribe Full Name"] < b["Tribe Full Name"] ){
+      return -1;
+    }
+    if ( a['Tribe Full Name'] > b['Tribe Last Name'] ){
+      return 1;
+    }
+    return 0;
+  }
+
+// process each line
+tribalLeaders.forEach(object => {
+    // remove OBJECTID
+    delete object['OBJECTID'];
+
+    //strip newlines from all but the last element
+    for(const property in object) {
+        if(property === 'y') {
+            break;
+        }
```

Comment:
I think you're right, but in my mind filter is for instances where you want to remove whole elements from a list based on some function. Since what I want to do here is alter data in-place, I think it's a bit more readable to just loop through it explicitly.